indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -206,6 +198,15 @@
 .abstract-review-badges {
     flex-grow: 1;
     color: $gray;
+
+    .js-ratings-toggle {
```

Comment:
It's funny how we have developed different understanding of `js-*` classes and I think we haven't been consistent about their usage. There are cases where we add an extra `js-*` class only to bind something to the element in JS. Maybe we should discuss it in the next meeting and write down the conclusion on the wiki.